tests/kernel/run_kselftests.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Executes kselftests.
+# This module introduces simplistic openqa runner for kernel selftests. The test
+# module allows running tests kernel selftests from either git repository which should be
+# defined in the test setting: KERNEL_GIT_TREE or from OBS/IBS repository using packaged
+# and build rpm. As of May-2025 this runner is meant exclusively for cgroup tests.
+# Running from git supports checking out specific git tag version of the kernel, so if required
+# the tests can checkout the older version, corresponding with the kernel under tests, and run
+# such tests
+#
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use base 'opensusebasetest';
+
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use registration;
+use utils;
+
+sub prepare_kselftests_from_git
+{
+    my ($root) = @_;
+
+    my $git_tree = get_var('KERNEL_GIT_TREE', 'https://github.com/torvalds/linux.git');
+    my $git_tag = get_var('KERNEL_GIT_TAG', 'none');
```

Comment:
nit: I would use just empty string (if `undef` does not work as you want).

Reply:
I changed to '' since udef - I think - is throwing some warnings. Not a big deal but if you are ok with '' I consider this resolved.